src/main/scala/com/reagroup/appliedscala/urls/fetchmovie/FetchMovieController.scala

```diff
@@ -19,7 +19,15 @@ class FetchMovieController(fetchMovie: MovieId => IO[Option[Movie]]) extends Htt
     * Hint: You can use `NotFound()` to construct a 404 and `Ok()` to construct a 200.
     * Delegate the error case to the `ErrorHandler`.
     */
-  def fetch(movieId: Long): IO[Response[IO]] =
-    ???
+  def fetch(movieId: Long): IO[Response[IO]] = {
```

Comment:
Answer Key did it like that
```scala
  def fetch(movieId: Long): IO[Response[IO]] = for {
    errorOrMovie <- fetchMovie(MovieId(movieId)).attempt
    resp <- errorOrMovie match {
      case Right(Some(movie)) => Ok(movie)
      case Right(None) => NotFound("Oops not found")
      case Left(e) => ErrorHandler(e)
    }
  } yield resp
```

Question:
How come answer key did not need to do `movie.asJson` ?

As json conversion implicit when you return a http4s response?????